### ISSUE TITLE
panic while getting logs for an app

### ISSUE THREAD

**Issue by clayzermk1** [deis/deis#4592]
deis cluster v1.10.1 with stable CoreOS v766.4.0

`deis --version` -> 1.10.1

I have an app that I have very verbose logging on.

`deis logs -a <app>`
```
<a small amount of logs that are chronologically old>
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/deis/deis/client/cmd.AppLogs(0x7fff5fbffc05, 0xe, 0xffffffffffffffff, 0x0, 0x0)
	/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/cmd/apps.go:162 +0x571
github.com/deis/deis/client/parser.appLogs(0x8206e4290, 0x3, 0x3, 0x0, 0x0)
	/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/parser/apps.go:192 +0x2b6
github.com/deis/deis/client/parser.Apps(0x8206e4290, 0x3, 0x3, 0x0, 0x0)
	/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/parser/apps.go:37 +0x3e9
main.Command(0x8206e4290, 0x3, 0x3, 0x186840)
	/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/deis.go:87 +0xc24
main.main()
	/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/deis.go:18 +0x60
```

**Comment by bacongobbler** [comment]
I believe this is a duplicate of https://github.com/deis/deis/pull/4420, which is fixed now in v1.11.0 :)

**Comment by clayzermk1** [comment]
It appears to be, thank you.

I swear I did a search in issues for "panic"...

Cheers guys :beers:!
